src/internal/async-store/index.ts

```diff
@@ -1,8 +1,8 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
+
 import { useLayoutEffect, useState } from 'react';
-import { unstable_batchedUpdates } from 'react-dom';
```

Comment:
The effect of using unstable_batchedUpdates is not clear - the added integration tests do not show any difference when run with and w/o it.